sql/wsrep_mysqld.cc

```diff
@@ -2870,6 +2870,57 @@ static void wsrep_RSU_end(THD *thd) {
   thd->disable_binlog_guard.reset();
 }
 
+bool thd_enter_apply_monitor(THD* thd) {
+
+  if (thd->killed != THD::NOT_KILLED) {
```

Comment:
shouldn't it be under if (thd->slave_thread)?

Reply:
Agree. Moved it to under `if (thd->slave_thread)`.